test/Microsoft.Health.Fhir.Shared.Tests.E2E/Rest/ValidateTests.cs

```diff
@@ -0,0 +1,155 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using Hl7.Fhir.Model;
+using Microsoft.Health.Fhir.Tests.Common;
+using Microsoft.Health.Fhir.Tests.Common.FixtureParameters;
+using Xunit;
+
+namespace Microsoft.Health.Fhir.Tests.E2E.Rest
+{
+    [Trait(Traits.Category, Categories.Validate)]
+    [HttpIntegrationFixtureArgumentSets(DataStore.All, Format.Json)]
```

Comment:
We might want to have at least a single test that will test the XML code path if validate is also enabled for it.

Reply:
Would that involve making another file for XML tests, or can I apply that trait attribute to individual tests?

Reply:
You can apply the trait to individual tests. See `WhenVersionsEndpointIsCalledWithXml_GivenAValidXmlAcceptHeaderIsProvided_ThenServerShouldReturnOK` for an example.

Reply:
Test added